internal/ingress/controller/template/crossplane/crossplane_test.go

```diff
@@ -63,14 +63,60 @@ func TestCrossplaneTemplate(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, mimeFile.Close())
 
-	t.Run("it should be able to marshall and unmarshall the current configuration", func(t *testing.T) {
-		tplConfig := &config.TemplateConfig{
-			Cfg: config.NewDefault(),
-		}
+	tplConfig := &config.TemplateConfig{
+		Cfg: config.NewDefault(),
+	}
+	tpl := crossplane.NewTemplate()
+
+	t.Run("it should be able to marshall and unmarshall the default configuration", func(t *testing.T) {
 		tplConfig.Cfg.DefaultSSLCertificate = defaultCertificate
 		tplConfig.Cfg.EnableBrotli = true
```

Comment:
this probably should be moved to the specifics configurations test below, but we can do on a follow up